src/blinx.js

```diff
@@ -157,6 +157,19 @@ let _callRender = function (module, placeholderResponse) {
 		}
 
 		res();
+
```

Comment:
This is more of module's logic. Lets move this to logic to module in a member function and we can call that function from here.
Let me know if you have some other approach in mind.